styleguide/derek/view-templates/leadership/_leadership.scss

```diff
@@ -0,0 +1,49 @@
+.leadership-individual {
+  @include make-sm-column(3);
+  @include make-xs-column(6);
+  margin: 20px auto;
+}
+
+.leadership-image {
+  @include center-block;
+}
+
+.leadership-attrWrapper {
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.leadership-name {
+  margin-bottom: 0;
+}
+
+.leadership-title {
+  margin: 0;
+}
+
+.leadership-linkWrapper {
+  margin-top: 20px;
+}
+
+.leadership-link {
+  @include standard-link;
+}
+
+@media only screen and (min-width: $screen-xs-max) {
+  .leadership-individual {
+    &.center-block {
+      float: none;
+    }
+  }
+}
+
+@media only screen and (max-width: $screen-sm-max) {
```

Comment:
I fixed that one. The one that handles the lone-wolf leaders though won't override the floats properly when I try to do mobile-first. Bootstrap says no.